canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -141,6 +141,22 @@ class PropertiesMain extends React.Component {
 		}
 	}
 
+	componentDidUpdate(prevProps) {
+		if (!isEqual(prevProps.callbacks, this.props.callbacks)) {
+			this.propertiesController.setHandlers({
```

Comment:
Can we create a new setCallbacks method that does this for both here and in the constructor?  That way when a new callback is added we only need to update 1 place.